src/utils/miscellaneous.ts

```diff
@@ -49,7 +49,11 @@ export function getToken<T extends TokenDetails, K extends keyof T>(
   value: string | number | undefined = '',
   tokens: T[] | undefined | null,
 ): T | undefined {
-  return (tokens || []).find((token: T): boolean => {
+  if (!tokens) {
```

Comment:
nit: probably cleaner to write it as `if( tokens) { ... })` returns `undefined` if no `tokens`